app/models/hud_reports/question_sheet.rb

```diff
@@ -13,7 +13,7 @@ class QuestionSheet
 
     def initialize(report:, question:)
       @report = report
-      raise "invalid question id #{question.inspect}" unless question =~ /\AQ\d+/
+      raise ArgumentError, "invalid question id #{question.inspect}" unless question =~ /\AQ\d+/
```

Comment:
added type to support unit tests

Reply:
Oh, I see, this isn't a change to the check, just what is raised.  Looks ok.